README.md

```diff
@@ -1,6 +1,5 @@
 # litentry-parachain
-[![parachain](https://github.com/litentry/litentry-parachain/actions/workflows/parachain-ci.yml/badge.svg)](https://github.com/litentry/litentry-parachain/actions/workflows/parachain-ci.yml)
-[![tee-worker](https://github.com/litentry/litentry-parachain/actions/workflows/tee-worker-ci.yml/badge.svg)](https://github.com/litentry/litentry-parachain/actions/workflows/tee-worker-ci.yml)
+[![general ci](https://github.com/litentry/litentry-parachain/actions/workflows/ci.yml/badge.svg)](https://github.com/litentry/litentry-parachain/actions/workflows/ci.yml)
```

Comment:
This workflow is run on feature branches as well. This badge will show status on `dev` or any branch ? In issue comment I suggested limiting branches to `dev` to make sure we only display status from it.

Reply:
You are right. Missed `branch=dev`.